src/current/v23.1/backup.md

```diff
@@ -143,19 +143,28 @@ The `system.users` table stores your users and their passwords. To restore your
 
 ## Performance
 
-The `BACKUP` process minimizes its impact to the cluster's performance by distributing work to all nodes. Each node backs up only a specific subset of the data it stores (those for which it serves writes), with no two nodes backing up the same data.
+The backup job process minimizes its impact to the cluster's performance with:
 
-`BACKUP`, like any read, cannot export a range if the range contains an [unresolved intent](architecture/transaction-layer.html#resolving-write-intents). While you typically will want bulk, background jobs like `BACKUP` to have as little impact on your foreground traffic as possible, it's more important for backups to actually complete (which maintains your [recovery point objective (RPO)](https://en.wikipedia.org/wiki/Disaster_recovery#Recovery_Point_Objective)). Unlike a normal read transaction that will block until any uncommitted writes it encounters are resolved, `BACKUP` will block only for a configurable duration before invoking priority to ensure it can complete on-time.
+- Distribution of work to all nodes. Each node backs up only a specific subset of the data it stores (those for which it serves writes), with no two nodes backing up the same data. Refer to the [Backup Architecture](backup-architecture.html) page for a detailed explanation of how a backup job works.
+- {% include_cached new-in.html version="v23.1" %} Integration with elastic CPU limiter by default, which helps to minimize the impact backups have on foreground traffic. This integration will limit the amount of CPU time used by a backup thereby allowing foreground SQL traffic to continue largely unaffected.
+
+A backup job, like any read, cannot export a range if the range contains an [unresolved intent](architecture/transaction-layer.html#resolving-write-intents). While you typically will want bulk, background jobs like `BACKUP` to have as little impact on your foreground traffic as possible, it's still important for backups to complete (which maintains your [recovery point objective (RPO)](https://en.wikipedia.org/wiki/Disaster_recovery#Recovery_Point_Objective)).
```

Comment:
can you clarify what is meant by "important for backups to complete"? As opposed to.. backups not completing?

Reply:
Hey @ashleykwtam, not quite sure what you mean here.. could you provide a suggestion for clarity?

Reply:
Adding the "it's still important for backups to complete" reads awkward to me, as it seems to imply that backups don't have to complete and it's not important to complete. 

Perhaps we can phrase it like:
Even though it's important to minimize the impact of bulk, background jobs like `BACKUP` on your foreground traffic, it is still crucial for backups to finish (in order to maintain your recovery point objective (RPO)).